src/App.tsx

```diff
@@ -1,156 +1,213 @@
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
-      </div>
+          </header>
+
+          <section className="todoapp__main">
+            {visibleTodos.map(todo => {
+              return (
+                <div className={classNames('todo', {
+                  completed: todo.completed,
+                })}
+                >
+                  <label className="todo__status-label">
+                    <input
+                      type="checkbox"
+                      className="todo__status"
+                      checked={todo.completed}
+                    />
+                  </label>
+
+                  <span className="todo__title">{todo.title}</span>
+
+                  {/* Remove button appears only on hover */}
+                  <button
+                    type="button"
+                    className="todo__remove"
+                    onClick={() => handleDeleteTodo(todo.id)}
+                  >
+                    ×
+
+                  </button>
+
+                  {/* overlay will cover the todo while it is being updated */}
+                  <div className="modal overlay">
+                    <div className="modal-background has-background-white-ter" />
+                    <div className="loader" />
+                  </div>
+                </div>
+              );
+            })}
+          </section>
+
+          {/* Hide the footer if there are no todos */}
+          {todos.length !== 0 && (
+            <footer className="todoapp__footer">
+              <span className="todo-count">
+                {`${visibleTodos.filter(todo => !todo.completed).length} items left`}
+              </span>
+
+              {/* Active filter should have a 'selected' class */}
+              <nav className="filter">
+                <a
+                  href="#/"
+                  className={classNames('filter__link', {
+                    selected: filter === 'all',
+                  })}
+                  onClick={() => setFilter('all')}
+                >
+                  All
+                </a>
+
+                <a
+                  href="#/active"
+                  className={classNames('filter__link', {
+                    selected: filter === 'active',
+                  })}
+                  onClick={() => setFilter('active')}
+                >
+                  Active
+                </a>
+
+                <a
+                  href="#/completed"
+                  className={classNames('filter__link', {
+                    selected: filter === 'completed',
+                  })}
+                  onClick={() => setFilter('completed')}
+                >
+                  Completed
+                </a>
+              </nav>
+
+              {/* don't show this button if there are no completed todos */}
+              <button type="button" className="todoapp__clear-completed">
```

Comment:
Add a condition to display the 'todoapp__clear-completed' button only if there are completed todos.